libs/designsystem/header/src/header.component.scss

```
@@ -0,0 +1,49 @@
+@use 'sass:map';
+@use '@kirbydesign/core/src/scss/utils';
+
+.container {
+  width: 100%;
+  padding: utils.size('s') utils.size('s') utils.size('l');
+  display: flex;
+  flex-direction: column;
+
+  @media screen and (min-width: map.get(utils.$breakpoints, 'medium')) {
+    padding-top: utils.size('m');
+  }
+
+  &.centered {
+    align-items: center;
+  }
+}
+
+.avatar {
+  margin-bottom: utils.size('s');
+}
+
+.flag {
+  margin-bottom: utils.size('xxxs');
+}
+
+.title {
+  margin-bottom: utils.size('xxxxs');
+}
+
+.key {
+  margin-bottom: utils.size('xxxxs');
+}
+
+.value {
```

Consider reducing number of css classes and selectors:
```suggestion
h3 {
```